vtex/omnishipping
# Fixes not showing required error when in geolocation for number field

#### What is the purpose of this pull request?
### Fixed

- Not showing required error when in geolocation for number field

#### What problem is this solving?
Closes [story](https://app.clubhouse.io/vtex/story/18562/campo-obrigat%C3%B3rio-n%C3%BAmero-sem-hint-tooltip-quando-endere%C3%A7o-%C3%A9-por-geolocation)

#### How should this be manually tested?
1. Add [items to cart](https://fernando--vtexgame1geo.myvtex.com/checkout/cart/add?&workspace=fernando&sku=298&qty=1&seller=1&sc=1)
2. Go to Shipping and type `Praia de botafogo` without the number
3. Select the first item of the list
4. Try to calculate shipping and it should show the required error below the number field.

#### Screenshots or example usage
n/a
#### Types of changes
